src/dotnet/APIView/APIViewWeb/Managers/CodeFileManager.cs

```diff
@@ -201,6 +201,11 @@ public async Task<APICodeFileModel> CreateReviewCodeFileModel(string apiRevision
         /// <returns></returns>
         public async Task<bool> AreAPICodeFilesTheSame(RenderedCodeFile codeFileA, RenderedCodeFile codeFileB)
         {
+            if (codeFileA.CodeFile.VersionString != codeFileA.CodeFile.VersionString)
```

Comment:
@chidozieononiwu I don't think this will fix. Call is failing before it reaches this point. It's failing when it tries to deserialize the old code file using new tree style.
https://github.com/Azure/azure-sdk-tools/blob/main/src/dotnet/APIView/APIViewWeb/Repositories/BlobCodeFileRepository.cs#L48